tests/unit/utils/network_test.py

```diff
@@ -106,8 +106,13 @@ def test_sanitize_host(self):
         self.assertEqual(ret, '10.1.2.3')
 
     def test_host_to_ips(self):
+        '''
+        NOTE: When this test fails it's usually because the IP address has
+        changed. In these cases, we just need to update the IP address in the
+        assertion.
+        '''
         ret = network.host_to_ips('www.saltstack.com')
-        self.assertEqual(ret, ['104.199.122.13'])
+        self.assertEqual(ret, ['104.197.168.128'])
```

Comment:
@terminalmage I find two problems here:
1. Unit test is using a real network. This should be actually done using mocking. That will allow them running in isolated build services where everything is offline. The real stuff I'd put to the integration tests instead.
2. If you still decided to actually go real network, then why to hard-code the IP address, instead of obtaining it and compare the result?

```python
import os
...
self.assertEqual(ret, [os.popen("nslookup www.saltstack.com | awk '/^Address: /{print $2}'").read().strip()]
```
...?

@cachedout, @Ch3LL thoughts?